buy_together_app/models.py

```diff
@@ -1,3 +1,51 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
+    user_name = models.CharField(max_length=32, primary_key=True, null=False)
+    first_name = models.CharField(max_length=32)
+    last_name = models.CharField(max_length=32)
+    password = models.CharField(max_length=16)
+    access = models.BooleanField()
```

Comment:
look at the ERD, access has been changed to user_type and the field is charfield

Reply:
last thing add description to the PR, "adding supplier product, tests etc..."